src/pathwayvis/components/reactionpanel/addedReactions/addedReactions.component.ts

```diff
@@ -60,10 +60,7 @@ class AddedReactionsController {
   }
 
   private updateMapData(response) {
```

Comment:
Why don't you call `this._mapOptions.updateMapData(response);` Then this method would be unnecessary.